### PR TITLE
Appoiement print 2

### PR DESCRIPTION
**Pull request by venkyp143**
Appoiement letter works

### CODE REVIEW

**Comment by venkyp143** [comment]
please merge

**Comment by venkyp143** [comment]
merges solved please check

**Comment by venkyp143** [comment]
please merge

File: ui/src/Components/Store/Store.js [hunk 1]
```diff
@@ -4,11 +4,13 @@ import { configureStore } from "@reduxjs/toolkit";
 import loginSlice  from "./LoginSlice";
 import offerletterReducer  from "./Offer-LetterSlice";
 import payslipReducer  from "./PayslipSlice";
+import appoiementReducer from "./AppoiementSlice";
 
 export default  configureStore({
     reducer :{
         login:loginSlice,
         payslip:payslipReducer,
+        appoiement: appoiementReducer,
```
Comment (TriNitSTechnologies) [inline ui/src/Components/Store/Store.js:13]: Change to 'appointment'

Comment (TriNitSTechnologies) [inline ui/src/Components/Store/Store.js:13]: Change to 'appointmentReducer'

File: ui/src/Components/Store/AppoiementSlice.js [hunk 2]
```diff
@@ -0,0 +1,23 @@
+import { createSlice } from "@reduxjs/toolkit"
+
+
+const initialState={
+    appoienment: {
+
+    }
+}
+
+export const  AppoiementSlice= createSlice({
```
Comment (TriNitSTechnologies) [inline ui/src/Components/Store/AppoiementSlice.js:10]: Change to 'AppointmentSlice'

**Review by TriNitSTechnologies — APPROVED** [review]

**Pull request merged**